### PR TITLE
Remove warnings in akka stream testkit module

### PR DESCRIPTION
**Pull request by chbatey**

### CODE REVIEW

File: akka-stream-testkit/src/main/scala/akka/stream/testkit/StreamTestKit.scala [hunk 1]
```diff
@@ -83,6 +83,7 @@ object TestPublisher {
 
     type Self <: ManualProbe[I]
 
+    @ccompatUsedUntil213
```
Comment (raboof) [inline akka-stream-testkit/src/main/scala/akka/stream/testkit/StreamTestKit.scala]: weird place for this annotation? but it doesn't matter where it goes so :+1: :smile:

**Review by raboof — APPROVED** [review]
sweet

**Comment by raboof** [comment]
PLS BUILD

**Review by johanandren — APPROVED** [review]
LGTM

**Pull request merged**